.github/workflows/publish.yml

```diff
@@ -67,9 +67,14 @@ jobs:
       run: npm run deploy:flutter
       working-directory: importer
 
-    # TODO: https://github.com/microsoft/fluentui-system-icons/issues/34
-    # - name: Build and publish Flutter library
-    #   uses: ?
+    # The name should be same as the package name on pub.dev
+    - name: 'fluentui_system_icons'
+      uses: k-paxian/dart-package-publisher@master
+      with:
+        relativePath: '././flutter'
```

Comment:
Do we need `././` or can this be just `flutter`?

Reply:
Updated. :)